modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterManagementGroupManager.java

```diff
@@ -593,6 +593,9 @@ private void destroyCmg() {
 
                 raftManager.stopRaftNodes(CmgGroupId.INSTANCE);
 
+                RaftNodeId nodeId = raftNodeId(new Peer(clusterService.nodeName()));
+                raftManager.destroyRaftNodeStorages(nodeId, raftGroupOptionsConfigurer);
```

Comment:
Will we also delete on recovery node?

Reply:
No, but this doesn't seem to be a problem for any of the init and repair scenarios

Reply:
Let's add tests then to make sure.

Reply:
There are tests for cancellation, and I'm developing the feature of CMG reparation, it will have its own tests.

Reply:
ok